content/sensu-go/6.2/operations/deploy-sensu/secure-sensu.md

```diff
@@ -210,22 +213,20 @@ The `Subject:` field indicates the certificate's CN is `client`, so to bind the
 
 To enable agent mTLS authentication:
 
-1. Create and distribute new certificates and keys according to the [Generate certificates][12] guide.
-
-2. With the TLS certificate and key are in place, [update the agent configuration using `cert-file` and `key-file` security configuration flags][7].
+1. Create and distribute a new Certificate Authority (CA) root certificate and new agent and backend certificates and keys according to the [Generate certificates][12] guide.
 
-3. After you create backend and agent certificates, modify the backend configuration:
+2. Add the following parameters and values to the backend configuration:
 {{< code yml >}}
-agent-auth-cert-file: "/path/to/backend-1.pem"
-agent-auth-key-file: "/path/to/backend-1-key.pem"
-agent-auth-trusted-ca-file: "/path/to/ca.pem"
+agent-auth-cert-file: "/etc/sensu/tls/backend-1.pem"
+agent-auth-key-file: "/etc/sensu/tls/backend-1-key.pem"
```

Comment:
Should the values for these flags be the paths to backend cert and key? Seems like it should be agent, but the descriptions and examples for these two flags in the backend reference and the section that addresses this in the agent reference are too vague to be helpful.